### PR TITLE
Add Options for Hide Password/License Key

### PR DESCRIPTION
**Pull request by alagoutte**
and also update Samples

### CODE REVIEW

**Comment by alagoutte** [comment]
hi @tpcarman or @rebelinux can you merge ?

**Comment by rebelinux** [comment]
I do not have privileges to do the merge in this repository :(

File: Src/Private/Get-AbrCPPMAuthentication.ps1 [hunk 1]
```diff
@@ -239,9 +239,15 @@ function Get-AbrCPPMAuthentication {
                                         foreach ($cpascd in $s.cppm_primary_auth_source_connection_details.PSObject.Properties) {
                                             #Only display when there is a value...
                                             if ($cpascd.value) {
+                                                $value = $cpascd.value
+                                                if ($Options.HidePassword) {
+                                                    if ($cpascd.name.contains("password") ) {
+                                                        $value = "ANONYMISED"
+                                                    }
```
Comment (tpcarman) [inline Src/Private/Get-AbrCPPMAuthentication.ps1]: Would this be better? 
```
if (($Options.HidePassword) -and ($cpascd.name.contains("password"))) {
	$value = "Anonymised"
} else {
	$value = $cpascd.value
}
```
Please refrain from using uppercase, use TitleCase instead.

File: Src/Private/Get-AbrCPPMLicense.ps1 [hunk 2]
```diff
@@ -39,16 +39,16 @@ function Get-AbrCPPMLicense {
                     $OutObj = @()
                     foreach ($item in $summary) {
                         $OutObj += [pscustomobject]@{
-                            "Type" = $item.license_type
-                            "Count" = $item.licensed_count
-                            "Used" = $item.used_count
+                            "Type"    = $item.license_type
+                            "Count"   = $item.licensed_count
+                            "Used"    = $item.used_count
```
Comment (tpcarman) [inline Src/Private/Get-AbrCPPMLicense.ps1:44]: Do you have something in your VS Code settings which is changing the alignment?

Comment (alagoutte) [reply Src/Private/Get-AbrCPPMLicense.ps1:44]: Yes, i use some other project/machine, where it is not alignPropertyValuePairs... and get sometimes issues ^^

**Review by tpcarman — APPROVED** [review]

**Pull request merged**